README.md

```diff
@@ -1,12 +1,16 @@
+# Deprecation Notice
+
+> [!CAUTION]
+> This community maintained project will be archived. For the fully supported Camunda Spring SDK, refer to the [docs](https://docs.camunda.io/docs/apis-tools/spring-zeebe-sdk/getting-started/). The Operate client was moved to a [dedicated repository](https://github.com/camunda-community-hub/camunda-operate-client-java).
```

Comment:
```suggestion
> This community-maintained project will be archived. 
* For the fully supported Camunda Spring SDK, refer to the [docs](https://docs.camunda.io/docs/apis-tools/spring-zeebe-sdk/getting-started/). 
* The community-maintained Operate API client was moved to a [dedicated repository](https://github.com/camunda-community-hub/camunda-operate-client-java).

In the future, Camunda will expand on fully supported Camunda Spring SDK to support [Camunda 8 REST API](https://docs.camunda.io/docs/next/apis-tools/camunda-api-rest/camunda-api-rest-overview/) for the entire orchestration cluster. 
```

Reply:
☝️ I made a very slight edit to @aleksander-dytko's suggestion.